.github/workflows/ci.yaml

```diff
@@ -52,25 +52,4 @@ jobs:
   deploy:
     if: github.repository == 'nextstrain/augur' && github.ref == 'refs/heads/release'
     needs: [test]
-    runs-on: ubuntu-latest
-    steps:
-      - env:
-          TRAVIS_AUTH_TOKEN: ${{ secrets.TRAVIS_AUTH_TOKEN }}
-        run: |
-          set -euo pipefail
-
-          echo "Pinging Travis CI to rebuild Docker image"
-
-          body='{
-            "request": {
-              "branch": "master",
-              "message": "Build triggered from augur"
-            }
-          }'
-
-          curl -X POST https://api.travis-ci.com/repo/nextstrain%2Fdocker-base/requests \
-            -H "Content-Type: application/json" \
-            -H "Accept: application/json" \
-            -H "Travis-API-Version: 3" \
-            -H "Authorization: token $TRAVIS_AUTH_TOKEN" \
-            -d "$body"
+    uses: nextstrain/docker-base/.github/workflows/ci.yml
```

Comment:
Calling the workflow in docker-base won't work, as the entire workflow/event context is the caller's, so will be Augur's. e.g. checkout in the called workflow will get Augur instead of docker-base. We could bend things to our will and make it work, but I think it's a confusing choice anyhow as it would spread the logs for automated image builds across multiple repos.

Instead, I'd suggest having this Augur workflow issue a `workflow_dispatch` event via GitHub's REST API, i.e. basically the same thing it does with Travis now but pointed at GitHub instead. I believe there's an example of this with ncov-ingest dispatching to ncov.

Reply:
Good point. I opted for [`gh workflow run`](https://github.com/nextstrain/docker-base/blob/master/.github/workflows/ci.yml) instead of REST API, but it should accomplish the same thing.

Reply:
Oh, nicer indeed. :-)